components/eamxx/CMakeLists.txt

```diff
@@ -189,6 +189,9 @@ if (SCREAM_DEBUG)
   endif ()
 endif()
 
+set(DEFAULT_RRTMGP_ENABLE_YAKL TRUE)
```

Comment:
Since these are hardcoded, and never change, you could prob just use TRUE/FALSE in the `option(...)` lines down below...